{"Desktop\\Work\\tensorflow-master\\tensorflow\\python\\saved_model\\save_context": ["Desktop\\Work\\tensorflow-master\\tensorflow\\python\\saved_model\\save_context.SaveContext.__init__", "contextlib.contextmanager"], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\saved_model\\save_context.SaveContext.__init__": ["<builtin>.super"], "<builtin>.super": [], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\saved_model\\save_context.SaveContext.options": ["<builtin>.ValueError", "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\saved_model\\save_context.SaveContext.in_save_context"], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\saved_model\\save_context.SaveContext.in_save_context": [], "<builtin>.ValueError": [], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\saved_model\\save_context.SaveContext.enter_save_context": [], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\saved_model\\save_context.SaveContext.exit_save_context": [], "contextlib.contextmanager": [], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\saved_model\\save_context.save_context": ["<builtin>.ValueError", "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\saved_model\\save_context.SaveContext.enter_save_context", "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\saved_model\\save_context.SaveContext.exit_save_context", "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\saved_model\\save_context.in_save_context"], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\saved_model\\save_context.in_save_context": ["Desktop\\Work\\tensorflow-master\\tensorflow\\python\\saved_model\\save_context.SaveContext.in_save_context"], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\saved_model\\save_context.get_save_options": ["Desktop\\Work\\tensorflow-master\\tensorflow\\python\\saved_model\\save_context.SaveContext.options"]}
